src/redux/actions/experimentSettings/processingConfig/loadedProcessingConfig.js

```diff
@@ -0,0 +1,25 @@
+import {
+  EXPERIMENT_SETTINGS_PROCESSING_CONFIG_LOADED,
+} from '../../../actionTypes/experimentSettings';
+
+import saveProcessingSettings from './saveProcessingSettings';
+
+const loadedProcessingConfig = (
+  experimentId, processingConfig, fromGem2s = false,
+) => (dispatch) => {
+  dispatch({
+    type: EXPERIMENT_SETTINGS_PROCESSING_CONFIG_LOADED,
+    payload: {
+      data: processingConfig,
+      fromGem2s,
+    },
+  });
+
+  if (fromGem2s) {
+    Object.keys(processingConfig).forEach((step) => {
+      dispatch(saveProcessingSettings(experimentId, step));
+    });
+  }
```

Comment:
why is this needed?

Reply:
This was added originally in a different PR than this one (different code but same thing, it sent updates to the api) when adding the `defaultFilterSettings` property in the ui on receiving processingConfig (it was in the `experimentUpdatesHandler`).

I think it should probably be moved to the api or (even better) to gem2s when generating the processingConfig, but I didn't want to delay the ticket more by doing something unrelated to it, so I just refactored it into the changes of my ticket.